src/uk/gov/hmcts/contino/YarnBuilder.groovy

```diff
@@ -101,4 +103,17 @@ EOF
   def yarn(task){
     steps.sh("yarn ${task}")
   }
+
+  def yarnQuiet(task){
+    return steps.sh(script: "yarn ${task} &> /dev/null", returnStatus: true)
+  }
+
+  def yarnWithCheck(task) {
```

Comment:
This doesn't read well to me,

can't we keep it the same:
```suggestion
  def yarn(task) {
```

and then make a private function, something like:
```
private runYarn(task) {
```